### DISCUSSION TITLE
Increase the summary length of a post in the home page - DoIt (Hugo Theme)

### DISCUSSION THREAD

**Discussion by soto-sergio** [HEIGE-PCloud/DoIt#910] [General]
I want to share with the DoIt community and for future reference.

1) To increase the length of a summary of a post in the homepage from 70 to _x_ amount:
 In Hugo's `config.toml` file, add the line  `summaryLength = "x"` with the _x_ amount of words desired.

2.  The next step consists in increasing the lines of text that are displayed in the box where the summary is contained.
Locate the following file and increase the value from 3 to _y_ amount of text lines.
https://github.com/HEIGE-PCloud/DoIt/blob/0ef5b0f03351ea033307be3af21704b7dc13e6b0/assets/css/_page/_home.scss#L127

* This line clamps the box size of the summary to three lines of text and increasing the amount will display more text.

**Comment by zc-devs** [comment]
For point 2 you should better use [styles customization](https://hugodoit.pages.dev/theme-documentation-basics/#style-customization):

```
/* assets/css/_custom.scss */

.home[posts] {
    .summary {
        .content {
            @include box(vertical);
            -webkit-line-clamp: 5;
        }
    }
}
```

**Reply by HEIGE-PCloud** [reply]
That line clamp limit is removed in a recent commit, so now you don't need to adjust CSS anymore :)

**Reply by zc-devs** [reply]
Great! Will drop it with next release.

(I came from LoveIt and currently using DoIt 0.4.0)
